scriptmodules/supplementary/runcommand/joy2key.py

```diff
@@ -30,6 +31,76 @@
 JS_EVENT_AXIS = 0x02
 JS_EVENT_INIT = 0x80
 
+RETROARCH_CFG = '/opt/retropie/configs/all/retroarch.cfg'
+DEFAULT_BTN_CODES = ['\n', '\t']
+
+def ini_get(key, cfg_file):
```

Comment:
OK. I've read [ConfigParser docs](https://docs.python.org/2/library/configparser.html),and some examples and then tried here. It expects `[Sections]` in the config files.

I googled about how to use ConfigParser with sectionless files and the solutions I found look very ugly. IMHO much uglier than implementing `ini_get()` here.

Do you know some way to use ConfigParser with sectionless files?